cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/startedCheckout.js

```diff
@@ -22,11 +24,20 @@ function getData(currentBasket) {
     data.Categories = [];
     data.Items = [];
     data.$email = currentBasket.customerEmail;
+    data.cartRebuildingLink = URLUtils.abs('KlaviyoRecreate-Cart').toString() + `?items=${reconstructCartItems}`;
```

Comment:
It looks like the value of reconstructCartItems here is an empty array. Should this be stringified and/or encoded before appending to the URL?